_includes/head.html

```diff
@@ -50,4 +50,8 @@
     ga('create', 'UA-73726852-1', 'auto');
     ga('send', 'pageview');
   </script>
+  <script>
+    var github_repo = "{{ site.repository | escape }}";
```

Comment:
Sounds good. Just make sure those variables are defined before you source github.js.